mteb/tasks/Classification/multilingual/NusaParagraphTopicClassification.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification, MultilingualTask
+
+_LANGUAGES = {
+    "btk": ["bbc-Latn"],
+    "bew": ["bew-Latn"],
+    "bug": ["bug-Latn"],
+    "jav": ["jav-Latn"],
+    "mad": ["mad-Latn"],
+    "mak": ["mak-Latn"],
+    "min": ["min-Latn"],
+    "mui": ["mui-Latn"],
+    "rej": ["rej-Latn"],
+    "sun": ["sun-Latn"],
+}
+
+
+class NusaParagraphTopicClassification(MultilingualTask, AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="NusaParagraphTopicClassification",
+        dataset={
+            "path": "gentaiscool/nusaparagraph_topic",
+            "revision": "abb43f8d5b9510b8724b48283aca26c4733eac5d",
+        },
+        description="NusaParagraphTopicClassification is a multi-class topic classification on 10 Indonesian languages.",
+        reference="https://github.com/IndoNLP/nusa-writes",
+        category="s2s",
+        type="Classification",
+        eval_splits=["train", "validation", "test"],
```

Comment:
```suggestion
        eval_splits=["test"],
```
seems like we can test with just the test set.